.travis.yml

```diff
@@ -78,6 +81,25 @@ matrix:
         - cmake -DCMAKE_C_COMPILER=clang-9 -DCMAKE_CXX_COMPILER=clang++-9 tests -DCMAKE_BUILD_TYPE=Release
         - make VERBOSE=1 && make test
         - make clean
+    
+    # -------------------- Clang 9 (Part 2) --------------------
+    - language: cpp
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:  # Install a newer GCC because https://stackoverflow.com/a/51512150/1007777.
+            - g++-9
+            - g++-9-multilib
+            - gcc-9-multilib
+            - linux-libc-dev:i386
+            - libc6-dev-i386
+            - libstdc++-7-dev:i386
+      script:
+        # Set up the toolchain.
+        - wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && sudo ./llvm.sh 9
+        - sudo apt install clang-tidy-9 clang-format-9
+        - clang++-9 -E -x c++ - -v < /dev/null    # Print the Clang configuration for troubleshooting purposes.
```

Comment:
Why?

Reply:
Additional tests were blocked by Travis because it exceeds the maximum time (50 minutes?). How do we go about this?

Reply:
We ran out of time again here https://travis-ci.org/github/UAVCAN/libcanard/jobs/731183819

Reply:
Sigh. Okay, how about removing `RELWITHDEBINFO`? It doesn't add much value. Maybe later we should migrate to a new CI provider that doesn't have such strict time limits.

Reply:
Okay, let's do that.